cmd/exporter/exporter.go

```diff
@@ -259,13 +262,20 @@ func main() {
 			klog.Fatalf("%s", fmt.Sprintf("failed to write response: %v", err))
 		}
 	})
+	if *enableProfEndpoint {
```

Comment:
how about we enable this by default and use a `disable` flag to opt out?

Reply:
not sure whether it has side effect? if it's for debug purpose I think better make it disable

Reply:
IMHO, we should just enable pprof by default. Most go based applications do it.

pprof has no or very little overhead. Its not run until the http endpoint is hit. The point of enabling it by default is to capture profiling details when kepler is struck  .. I.E. restart is not an option because it means the particular state that kepler is in may not be easy to reproduce. 

As mentioned above , please go through https://go.dev/src/net/http/pprof/pprof.go . 
The overhead isn't much at all, thus, lets enable pprof by default.

Reply:
#1383 enables it by default.